runtime/lua/vim/lsp/handlers.lua

```diff
@@ -279,6 +279,17 @@ RCS[ms.workspace_symbol] = response_to_list(util.symbols_to_items, 'symbols', fu
   return string.format("Symbols matching '%s'", ctx.params.query)
 end)
 
+--- @deprecated remove in 0.13
```

Comment:
What's the point of adding a method that's deprecated out of the bat? @lewis6991 understands the logic here better but I don't think we need to add another handler.

Reply:
We should not add the handler here. It should be added to `buf.lua` and not exposed.